apps/src/templates/instructions/InstructionsCsfRightCol.jsx

```diff
@@ -82,11 +81,7 @@ class InstructionsCsfRightCol extends React.Component {
   }
 
   getColumnHeight() {
-    const collapseButtonHeight = getOuterHeight(this.collapser, true);
-    const scrollButtonsHeight = this.scrollButtons
-      ? this.scrollButtons.getMinHeight()
-      : 0;
-    return collapseButtonHeight + scrollButtonsHeight;
+    return 0;
```

Comment:
Maybe add a comment indicating why returning 0 is the correct / ok thing to do here?